apps/desktop-wallet/src/features/transactionsDisplay/transactionLists/lists/WalletTransactionsList.tsx

```diff
@@ -127,7 +128,8 @@ const applyFilters = ({
         if (!txRefAddress) return false
 
         const { tokenAmounts } = calcTxAmountsDeltaForAddress(tx, txRefAddress)
-        const infoType = getTransactionInfoType(tx, txRefAddress)
+        const isInternalTransfer = isInternalTx(tx, allAddressHashes)
+        const infoType = getTransactionInfoType(tx, txRefAddress, isInternalTransfer)
```

Comment:
Should `getTransactionInfoType` take `allAddressHashes` as a third argument instead of `isInternalTransfer`?
The 2 places we uses this we anyway have `allAddressHashes` available. 
`isInternalTx` can be called withing `getTransactionInfoType` making the use in components a bit cleaner.
WDYT? :)